packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/mfaoob/setup-app.md

```diff
@@ -0,0 +1,28 @@
+4. Select **Native Application** as the **Application type**, then click **Next**.
+5. Specify the **App integration name**.
+6. Enable the **MFA OOB** grant type in addition to the defaults.
+7. Select **Allow everyone in your organization to access**, then click **Save**.
+8. From the **General** tab of your app integration, copy and save the generated **Client ID** value to implement your authorization flow.
+
+## Set up the authentication policy
```

Comment:
For the 2 MFA grants there is another way to configure the policies, which I think will be more common, although maybe we should lay out both approaches. The approach I think that will be more common is having the Global Session Policy require password and the app sign on policy requiring the additional factor